packages/env/package.json

```diff
@@ -35,10 +35,13 @@
 		"chalk": "^2.4.2",
 		"copy-dir": "^1.2.0",
 		"docker-compose": "^0.22.2",
+		"extract-zip": "^1.6.7",
 		"inquirer": "^7.0.4",
 		"js-yaml": "^3.13.1",
 		"nodegit": "^0.26.2",
 		"ora": "^4.0.2",
+		"request": "^2.88.2",
```

Comment:
Noting that `request` is deprecated, even at the time it was introduced here:

https://www.npmjs.com/package/request
https://github.com/request/request/issues/3142

Which isn't necessarily to say that a deprecation is bad, but we can't expect any future maintenance (which could become problematic for security issues and the like). Another option like [`got`](https://www.npmjs.com/package/got), [`bent`](https://github.com/mikeal/bent) (from the author of `request`), or just straight [`https`](https://nodejs.org/api/https.html) may be a more reliable long-term dependency. For the specific needs of progress events, the first of these alternatives does have the most compatible API for how we're using it here, though a combination of an `https` response `Content-Length` header plus an incrementing value in a [stream iteration](https://github.com/WordPress/gutenberg/blob/307e1fb2fc4955f77af8089ab380d457c71a20c2/packages/url/scripts/download-wpt-data.js#L47-L51) may suffice as well.

Reply:
Thanks for the heads up. Created an issue here: https://github.com/WordPress/gutenberg/issues/21336